scripts/run_ne13.jl

```diff
@@ -1,5 +1,5 @@
-using ParallelTemperingMonteCarlo
-using Random,Plots
+using ParallelTemperingMonteCarlo#update/version_2a
+using Random , BenchmarkTools, Plots
```

Comment:
BenchmarkTools use of it should be removed

Reply:
Done